doc/3/controllers/document/index.md

```diff
@@ -0,0 +1,58 @@
+---
+code: true
+type: page
+title: create
+description: Creates a new document
+---
+
+# create
+
+Creates a new document in the provided index and collection.
+
+---
+
+## Arguments
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> create(
+      final String index,
+      final String collection,
+      final ConcurrentHashMap<String, Object> document)
+throws NotConnectedException, InternalException
+
+public CompletableFuture<ConcurrentHashMap<String, Object>> create(
+      final String index,
+      final String collection,
+      final ConcurrentHashMap<String, Object> document,
+      final ConcurrentHashMap<String, Object> options)
+throws NotConnectedException, InternalException
+```
+
+| Arguments          | Type                                         | Description                       |
+| ------------------ | -------------------------------------------- | --------------------------------- |
+| `index`            | <pre>string</pre>                            | Index                             |
+| `collection`       | <pre>string</pre>                            | Collection                        |
+| `content`          | <pre>ConcurrentHashMap<String, Object></pre> | Content of the document to create |
```

Comment:
Should be document, as in the signature and the options should also be present in the tab